src/plugins/badge.coffee

```diff
@@ -584,6 +597,8 @@ class Badge
 
     stars
 
+  _displayStars: (rating) -> if rating is 0 then false else true
```

Comment:
What if rating is 0.2? We should show any stars. Maybe we should bypass the `_ratingToStars `.